src/domain/safe/entities/multisig-transaction.entity.ts

```diff
@@ -16,7 +16,7 @@ export type MultisigTransaction = {
   data: string | null; // TODO will be added under https://github.com/5afe/safe-client-gateway-nest/pull/132
   dataDecoded: any | null;
   ethGasPrice: string | null;
-  executionDate: Date;
+  executionDate: Date | null;
```

Comment:
This was changed since the transactions in the queue have a `null` `executionDate`. @fmrsabino I could move this to a separate PR if you prefer.